### PR TITLE
fix:The delete icon should have three states .and refactor: ```Overview.razor```，```AppModal.razor```，```ProjectModal.razor``` pages

### PR DESCRIPTION
**Pull request by MapleWithoutWords**

### CODE REVIEW

File: src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor [hunk 1]
```diff
@@ -1,15 +1,15 @@
 ﻿@inherits PmCompontentBase
 
-<SSheetDialog EnableDomReload ValueChanged="AppModalValueChanged" Value="_appFormModel.Visible" Title="@(_appFormModel.HasValue ? T("Edit App"):T("Add App"))" Icon="mdi-circle" IconColor="success">
+<SSimpleModal ValueChanged="AppModalValueChanged" Value="_appFormModel.Visible" Title="@(_appFormModel.HasValue ? T("Edit App"):T("Add App"))" Width="770" Height="@(_appFormModel.HasValue ? 1051:881)">
```
Comment (MayueCif) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/AppModal.razor:3]: Fixed height？

File: src/Web/MASA.PM.Web.Admin/Pages/Home/ProjectModal.razor [hunk 2]
```diff
@@ -1,15 +1,15 @@
 ﻿@inherits PmCompontentBase
 
-<SSheetDialog EnableDomReload Value="_projectFormModel.Visible" ValueChanged="ProjectModalValueChanged" Title="@(_projectFormModel.HasValue ? $"{T("Edit Project")}":$"{T("Add Project")}")" Icon="mdi-circle" IconColor="success">
+<SSimpleModal Value="_projectFormModel.Visible" ValueChanged="ProjectModalValueChanged" Title="@(_projectFormModel.HasValue ? $"{T("Edit Project")}":$"{T("Add Project")}")" Width="770" Height="@(_projectFormModel.HasValue ? 903:773)">
```
Comment (MayueCif) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/ProjectModal.razor:3]: height?

File: src/Web/MASA.PM.Web.Admin/Pages/Home/Overview.razor [hunk 3]
```diff
@@ -127,43 +129,53 @@
...
                 </div>
             </div>
         </MForm>
-    </SSheetDialog>
+    </ChildContent>
+</SSimpleModal>
 
-    <SSheetDialog EnableDomReload ValueChanged="ClusterModalValueChanged" Value="_clusterFormModel.Visible" Title="@(_clusterFormModel.HasValue ? $"{T("Edit Cluster")}":$"{T("Add Cluster")}")" Icon="mdi-circle" IconColor="success">
+<SSimpleModal ValueChanged="ClusterModalValueChanged" Value="_clusterFormModel.Visible" Title="@(_clusterFormModel.HasValue ? $"{T("Edit Cluster")}":$"{T("Add Cluster")}")" Width="770" Height="@(_clusterFormModel.HasValue ? 677:521)">
+    <DeleteContent>
```
Comment (MayueCif) [inline src/Web/MASA.PM.Web.Admin/Pages/Home/Overview.razor:169]: ?

**Review by MayueCif — COMMENTED** [review]
@MapleWithoutWords

**Comment by MapleWithoutWords** [comment]
> @MapleWithoutWords

image
设计图是这样的，需要改成小弹窗

**Comment by MapleWithoutWords** [comment]
effect image:
image
image
image
image
image
image

**Review by MayueCif — APPROVED** [review]

**Pull request merged**